src/main/java/de/uhd/ifi/se/decision/management/jira/model/git/CommentStyleType.java

```diff
@@ -0,0 +1,65 @@
+package de.uhd.ifi.se.decision.management.jira.model.git;
+
+public enum CommentStyleType {
+	NONE, JAVA_C, PYTHON, HTML;
+
+	public static CommentStyleType getCommentStyleTypeByName(String name) {
+		if (name == null || name.isBlank()) {
+			return NONE;
+		}
+		for (CommentStyleType commentStyleType : values()) {
+			if (commentStyleType.name().equalsIgnoreCase(name)) {
+				return commentStyleType;
+			}
+		}
+		return NONE;
+    }
+
+    public String getSingleLineCommentChar() {
+        switch (this) {
```

Comment:
Instead of these switch case statement, please work with enum properties as it is done in the DocumentationLocation (identifier) and LinkType enums, for example.